### PR TITLE
DataGrid: update Rendering Optimzation + How To sections

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization/30 Use repaintChangesOnly.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Enable the [repaintChangesOnly]() property to optimize performance in the cases below:
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization/30 Use repaintChangesOnly.md]:
```suggestion
Enable the [repaintChangesOnly]() property to optimize performance in the following cases:
```

File: concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization/30 Use repaintChangesOnly.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+- If you enable editing and use the **columns**.[setCellValue](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#setCellValue) method.
+
+- If a velocity of real-time updates is high. Refer to the following demo for an example: [Real Time Updates](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/RealTimeUpdates).
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization/30 Use repaintChangesOnly.md]: What do you mean by `a velocity`?

**Review by dirk-pieterse — APPROVED** [review]

File: concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization/30 Use repaintChangesOnly.md (hunk 2 above)
Comment (vladaskorohodova) [reply concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization/30 Use repaintChangesOnly.md]: I mean "speed". Should I change it?

Comment (vladaskorohodova) [reply concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization/30 Use repaintChangesOnly.md]: Or do you mean that 'velocity' is uncountable?

Comment (dirk-pieterse) [reply concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization/30 Use repaintChangesOnly.md]:
```suggestion
- If the speed of real-time updates is high. Refer to the following demo for an example: [Real Time Updates](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/RealTimeUpdates).
```

**Pull request merged**